# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/openoffice/expat-use-dmake.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./expat/expat.patch.orig	2005-05-23 13:57:24.000000000 +0200
+++ ./expat/expat.patch	2005-05-23 14:04:34.000000000 +0200
@@ -66,8 +66,8 @@
 ! #
 ! #*************************************************************************
 ! target:
-! 	+cd xmltok && $(MAKE) $(MFLAGS) $(CALLMACROS)
-! 	+cd xmlparse && $(MAKE) $(MFLAGS) $(CALLMACROS)
+! 	+cd xmltok && $(DMAKE) $(MFLAGS) $(CALLMACROS)
+! 	+cd xmlparse && $(DMAKE) $(MFLAGS) $(CALLMACROS)
 *** misc/expat/xmlparse/makefile.mk	Mon Apr 29 18:59:16 2002
 --- misc/build/expat/xmlparse/makefile.mk	Mon Apr 29 17:49:05 2002
 ***************
